pulp_ansible/app/migrations/0028_collectionversion_requires_ansible.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.17 on 2020-12-09 20:22
+
+import tarfile
+import yaml
+from django.db import migrations, models
+
+
+def set_requires_ansible(apps, schema_editor):
+    CollectionVersion = apps.get_model("ansible", "CollectionVersion")
+    for collection_version in CollectionVersion.objects.all():
+        artifact = collection_version.contentartifact_set.get().artifact
+        with artifact.file.open() as artifact_file, tarfile.open(fileobj=artifact_file, mode="r") as tar:
+            try:
+                file_obj = tar.extractfile("meta/runtime.yml")
+            except KeyError:
+                continue
+
+            meta_runtime = yaml.safe_load(file_obj)
+            if meta_runtime.get("requires_ansible"):
+                collection_version.requires_ansible = meta_runtime.get("requires_ansible")
+                collection_version.save()
```

Comment:
we can merge this with the data migration from https://github.com/pulp/pulp_ansible/pull/468